browser/iframe-browser.js

```diff
@@ -0,0 +1,106 @@
+define(function (require, exports, module) {
+    "use strict";
+
+    var CommandManager      = brackets.getModule("command/CommandManager"),
+        Menus               = brackets.getModule("command/Menus"),
```

Comment:
Am I supposed to be looking at a line specifically?

I do need:
var CommandManager      = brackets.getModule("command/CommandManager")
though, because I use one of its methods to bring up the split view